lbry/wallet/server/db/trending/variable_decay.py

```diff
@@ -60,185 +59,331 @@ def check_trending_values(connection):
             break
 
     if needs_reset:
-        print("Resetting some columns. This might take a while...", flush=True, end="")
+        print("Resetting some columns. This might take a while...", flush=True,
+              end="")
         c.execute(""" BEGIN;
                       UPDATE claim SET trending_group = 0;
                       UPDATE claim SET trending_mixed = 0;
```

Comment:
why not:
```
UPDATE claim SET trending_group = 0, trending_mixed = 0;
```
this should double the execution speed

Reply:
Okay. This basically never happens though.